pkg/services/object/put/distributed.go

```diff
@@ -178,7 +178,7 @@ func (t *distributedTarget) saveObject(obj objectSDK.Object, objMeta object.Cont
 
 	repRules := t.containerNodes.PrimaryCounts()
 	ecRules := t.containerNodes.ECRules()
-	if typ := obj.Type(); len(repRules) > 0 || typ == objectSDK.TypeTombstone || typ == objectSDK.TypeLock {
+	if typ := obj.Type(); len(repRules) > 0 || typ == objectSDK.TypeTombstone || typ == objectSDK.TypeLock || typ == objectSDK.TypeLink {
```

Comment:
Links can have quite some payload. Although typically I'd expect <10GB objects and that are just ~6K. Still, consider 60M link, can it be split?